platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/PkiTools.java

```diff
@@ -52,6 +52,11 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(PkiTools.class);
 
+  private static final Pattern TUPLE_PATTERN = Pattern.compile(".*[=].*");
```

Comment:
This could match things like `a=b=c=d` which would be invalid.

Reply:
Going to address this as technical debt.